package.json

```diff
@@ -1,13 +1,13 @@
 {
-    "name": "poker-player-javascript",
-    "description": "Javascript player from lean poker. http://leanpoker.org",
-    "version": "0.0.2",
-    "engines": {
-        "node": "<4"
-    },
-    "private": true,
-    "dependencies": {
-        "express": "4.x",
-        "body-parser": "1.15.2"
-    }
+  "name": "poker-player-javascript",
+  "description": "Javascript player from lean poker. http://leanpoker.org",
+  "version": "0.0.2",
+  "engines": {
+    "node": "6.1.5"
```

Comment:
This version specification is too specific and will not allow hotfixes / security fixes, what about something more general like 6.1.x or even 6.x (to allow non-breaking feature releases), ([more info](https://docs.npmjs.com/misc/semver#ranges))